packages/cactus-plugin-ledger-connector-daml/src/main/typescript/plugin-ledger-connector-daml.ts

```diff
@@ -0,0 +1,301 @@
+import type { Express } from "express";
+import OAS from "../json/openapi.json";
+
+import {
+  GetPartiesInvolvedEndpointRequest,
+  QueryIOUEndpointRequest,
+  QueryRawIOUEndpointRequest,
+  CreateIOUEndpointRequest,
+  CreateIOUEndpointResponse,
+  ExerciseIOUEndpointRequest,
+  ExerciseIOUEndpointResponse,
+} from "./generated/openapi/typescript-axios";
+
+import {
+  IPluginLedgerConnector,
+  IWebServiceEndpoint,
+  IPluginWebService,
+  ICactusPluginOptions,
+  ConsensusAlgorithmFamily,
+  ICactusPlugin,
+} from "@hyperledger/cactus-core-api";
+import { consensusHasTransactionFinality } from "@hyperledger/cactus-core";
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+
+import {
+  ICreateIOUEndpointOptions,
+  CreateIOUEndpoint,
+} from "./web-services/create-iou-endpoint";
+
+import {
+  IExerciseIOUEndpointOptions,
+  ExerciseIOUEndpoint,
+} from "./web-services/exercise-iou-endpoint";
+
+import {
+  IQueryIOUEndpointOptions,
+  QueryIOUEndpoint,
+} from "./web-services/query-iou-endpoint";
+
+import {
+  IQueryRawIOUEndpointOptions,
+  QueryRawIOUEndpoint,
+} from "./web-services/query-raw-iou-endpoint";
+
+import {
+  IGetPartiesInvolvedEndpointOptions,
+  GetPartiesInvolvedEndpoint,
+} from "./web-services/get-parties-involved-endpoint";
+
+import fetch from "node-fetch";
```

Comment:
@raynatopedrajeta Please use `axios` instead of node-fetch just to keep the cognitive load minimized for everybody (axios is well used already)